### PR TITLE
#92 add talks to member page

### PR DESCRIPTION
**Pull request by denys-medynskyi**

### CODE REVIEW

File: app/controllers/members_controller.rb [hunk 1]
```diff
@@ -1,13 +1,17 @@
 class MembersController < ApplicationController
-  helper_method :member, :members
+  helper_method :member, :members, :talks
 
   private
 
+  def talks
```
Comment (sveredyuk) [inline app/controllers/members_controller.rb]: member_talks (more declarative, just talks - it's like all talks, might be confusing)

Comment (denys-medynskyi) [reply app/controllers/members_controller.rb]: true

File: app/controllers/members_controller.rb [hunk 2]
```diff
@@ -1,8 +1,12 @@
 class MembersController < ApplicationController
-  helper_method :member, :members
+  helper_method :member, :members, :member_talks
```
Comment (denys-medynskyi) [inline app/controllers/members_controller.rb:2]: @sveredyuk fixed

**Comment by sveredyuk** [comment]
Good job

**Review by sveredyuk — APPROVED** [review]

**Pull request merged**